client/src/providers/Patient/initialState.js

```diff
@@ -0,0 +1,77 @@
+const initialState = {
+  patientInfo: {
+    data: {},
+    history: [],
+    editDialog: false,
+    historyDialog: false,
+  },
+  adminNotes: {
+    data: [],
+    editForm: false,
+    historyDialog: false,
+  },
+  forms: {
+    data: [],
+    viewDialog: false,
+    expandDialog: false,
+  },
+  handouts: {
+    data: [],
+    newDialog: false,
+    expandDialog: false,
+  },
+  documents: {
+    data: [],
+    newDialog: false,
+    expandDialog: false,
+  },
+  encounters: {
+    data: [],
+    newDialog: false,
+    expandDialog: false,
+  },
+  medicalNotes: {
+    data: [],
+    editForm: false,
+    historyDialog: false,
+  },
+  allergies: {
+    data: [],
+    newDialog: false,
+    expandDialog: false,
+  },
+  messages: {
+    data: [],
+    newDialog: false,
+    expandDialog: false,
+  },
+  requisitions: {
+    data: [],
+    newDialog: false,
+    expandDialog: false,
+  },
+  tests: {
```

Comment:
`tests` seems too generic. Can we take another name? like: `patientTests` or similar..

Reply:
Yes, we can do that.